locales/en-US/app.ftl

```diff
@@ -153,6 +153,7 @@ experimentGradReportPendingTitle = This experiment has ended
 experimentGradReportPendingCopy = We are working on a full report. Check back soon for the details.
 experimentGoToLink = Go to { $title }
 startedDateLabel = Experiment Start Date: <b>{$startedDate}</b>
+learnMoreLink = LEARN MORE >>
```

Comment:
cc @flodolo

Reply:
@meandavejustice this should probably be just `Learn More`. All caps and random symbols are bad news for l10n.

Reply:
All caps is fine, as long as you do it directly in the string and not via CSS.
As for the symbols, they're normally done via CSS too (:after), with some attention to RTL

Reply:
I can remove the all caps css rule, thanks for the heads up. I'll also move the `>>` into a pseudo selector. thanks :)